sopel/config/core_section.py

```diff
@@ -216,3 +216,16 @@ def homedir(self):
 
     verify_ssl = ValidatedAttribute('verify_ssl', bool, default=True)
     """Whether to require a trusted SSL certificate for SSL connections."""
+
+    bucket_burst_tokens = ValidatedAttribute('bucket_burst_tokens', int,
+                                             default=4)
+    """How many messages can be sent in burst mode."""
+
+    bucket_refill_rate = ValidatedAttribute('bucket_refill_rate', int,
+                                            default=1)
+    """How many tokens/second to add to the token bucket."""
+
+    bucket_empty_wait = ValidatedAttribute('bucket_empty_wait', float,
+                                           default=0.7)
+    """How long to wait before sending a messaging when not in burst
+    mode."""
```

Comment:
I think these should be prefixed with `flood_` or something, and moved to the correct alphabetical sort location in the file. These names are much easier to work with, and still pretty clear:

  - `flood_burst_lines`
  - `flood_refill_rate`
  - `flood_empty_wait`

Also, this particular docstring has a typo (`s/messaging/message/`).